### PR TITLE
Added timestamps to the extracted comment and thread info

### LINKED ISSUE #27: Please consider giving full date time
Hello,

Please can you consider simplifying:

``` r
timestamp_to_date <- function(timestamp) timestamp |>
  as.POSIXct(origin="1970-01-01") |>
  as.Date() |>
  format("%Y-%m-%d")
```

To 

``` r
timestamp_to_date <- function(timestamp) timestamp |>
  as.POSIXct(origin="1970-01-01")
```

So that we can have more fine-grained comparisons between comment dates (and times)? Or add an option to not simplify to dates?

Thank you.

Edit - I forgot to say, your comment at the end of issue #4, doesn't quite solve this issue because converting back to timestamp from a date, when the time has been dropped, doesn't give the original timestamp.

Here the "round trip" works
```r
timestamp_to_date <- function(timestamp) timestamp |>
  as.POSIXct(origin = "1970-01-01") 

as.numeric(as.POSIXct(timestamp_to_date(1644944045), origin = "1970-01-01"))
[1] 1644944045
```
But here it doesn't
``` r
timestamp_to_date <- function(timestamp) timestamp |>
  as.POSIXct(origin="1970-01-01") |>
  as.Date() |>
  format("%Y-%m-%d")

as.numeric(as.POSIXct(timestamp_to_date(1644944045), origin = "1970-01-01"))
[1] 1644883200
```

### PR DESCRIPTION
**Pull request by py9mrg**
# Added timestamps to the extracted comment and thread info
  
Hello Ivan,

I've added the timestamps to all the extracted comment and thread data. I also updated the tests accordingly, and bumped the version in DESCRIPTION and NEWS.md. I left cran-comments.md alone as I thought you probably want to test your end before messing with CRAN submission.

I added the timestamps immediately after the `date_utc` entries as it seemed to make sense to put them adjacent. Everything seems to work ok on some trial runs locally.

Let me know if that's ok for you.

Cheers,

Closes #27 
  
- [x] Tested locally on Reddit data
- [x] Updated version
- [ ] Updated docs
- [x] Updated NEWS.md
- [ ] Updated cran-comments.md

### CODE REVIEW

**Comment by ivan-rivera** [comment]
Looking great, thanks for the PR! Just one small thing: I suggest updating cran-comments.md with notes, warnings and errors (if any). Considering that you are running those checks locally, you should have that information available to you and it also means that I won't need to push any changes after merging this request -- I'll simply be able to pull and submit the results to CRAN. Unfortunately, the CRAN submission happens manually (I've investigated whether or not it's possible to automate it with no success). Other than that, the checks seem to be passing, so we can merge it as soon as the CRAN comments are updated.

Thanks again for your contribution!

**Review by ivan-rivera — APPROVED** [review]

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: RedditExtractoR
 Type: Package
 Title: Reddit Data Extraction Toolkit
-Version: 3.0.5
+Version: 3.0.6
```
Comment (ivan-rivera) [inline DESCRIPTION:4]: We also need to update the date field on line 10 in this file, just set that to today's date otherwise CRAN will complain

Comment (py9mrg) [reply DESCRIPTION:4]: Oh yeah, duh, sorry - that was obvious! Commit 5ad5cad on my fork has this updated. But I've never been involved in a workflow before so no idea how to merge it to this PR. I've googled and clicked everything but can't seem to work it out sorry.

Comment (py9mrg) [reply DESCRIPTION:4]: Hang on, it seems like it has done it automatically. That's pretty clever.

**Comment by ivan-rivera** [comment]
That's it, we are done! The package should be on CRAN in a few days, I've just submitted it. Thanks again for your contribution. If you have any more suggestions or would like to make any more changes, you are most welcome to contribute more :)

**Comment by py9mrg** [comment]
Great! Thank you for walking me through it, as you can tell it was a bit of a learning curve!

**Pull request merged**